schema/changelog-4.4.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd"
+  logicalFilePath="changelog-4.4">
+
+  <changeSet author="author" id="changelog-4.4-all">
+    <addColumn tableName="tc_users">
```

Comment:
I don't remember if we discussed this, but can we use attributes for this instead?

Reply:
This was not discussed before yet.  
The enable totp could be an attribute. 
The totp key field is server side generated. And there needs to be an option to clear/regenerate it. I do not know if this is possible with the attributes. If so do you have an attribute which I can use as an example?

Reply:
I believe using attribute in this case is similar to NotificatorFirebase.

Reply:
I will have a look at this class and report back

Reply:
I think it's nice to use attributes here because it's optional functionality.

Reply:
What to do with the situation where the admin needs to enforce the use of it so it is not optional for a user? In our case the user should not be able to change it only the admin.

Reply:
I don't see any case where admin would need to enforce it on user by user basis.

Reply:
@tananaev no update on this?

Reply:
Waiting for response, as you can see.